3.6/Modules/mol_utils.py

```diff
@@ -70,17 +70,22 @@ def spf(mol, split_id):
 
 
 def create_chain(splits):
-    splits_ids = np.asarray(
-        [sorted([a.GetAtomicNum() for a in m.GetAtoms()
-              if a.GetAtomicNum() >= MOL_SPLIT_START]) for m in splits])
+    splits_ids = [x for x in [sorted([a.GetAtomicNum() for a in m.GetAtoms()
```

Comment:
I'm holding off with making the nsarray due to the the above mentioned can't assign an empty list to a nparray of a different size.

I think this is happening when the fragmentation is not working for a given molecule.